packages/web-core/src/services/WebAuthnService.ts

```diff
@@ -42,9 +42,28 @@ export class WebAuthnService {
     serializedChallenge: string,
     conditional: boolean,
     skipIfOnlyHybrid = false,
+    ac?: AbortController,
   ): Promise<Result<string, CorbadoError>> {
     try {
-      const abortController = this.abortOngoingOperation();
```

Comment:
If possible I would stick with one abort controller that controls the passkey operation (every abortController adds quite a bit of complexity).
We already use a callback in the react package to expose the abortController. Can we do this here in web-core as well? Instead of passing an ac we would pass a callback that accepts an ac.